doc/flame/structure.md

```diff
@@ -1,14 +1,16 @@
 # Structure
 
-Flame has a proposed structure for your project that includes the standard Flutter `assets` directory in addition to two children: `audio` and `images`.
+Flame has a proposed structure for your project that includes the standard Flutter `assets` 
+directory in addition to two children: `audio` and `images`.
 
 If using the following example code:
-
 ```dart
+void main() {
```

Comment:
Is it complaining without `main` here?

Reply:
Yes, it complains in pretty much every snippet, I just didn't have a stomach to fix them all :)
I think the analyzer in general only accepts snippets that can be valid dart files. So, since a statement like `FlameAudio.play('explosion.mp3');` can only appear inside a function/method, it won't be happy until there's some function there.